### ISSUE TITLE
No log colors when using winston

### ISSUE THREAD

**Issue by xiulunlin** [Unitech/PM2#1603]
Without pm2

<img width="658" alt="screen shot 2015-09-10 at 6 52 14 pm" src="https://cloud.githubusercontent.com/assets/7201022/9786508/1fd92d3a-57ed-11e5-9cb2-9411e930c5c4.png">

`pm2 logs --raw`

<img width="436" alt="screen shot 2015-09-10 at 6 52 26 pm" src="https://cloud.githubusercontent.com/assets/7201022/9786519/3b751efa-57ed-11e5-8c49-f1d257a519cb.png">

**Comment by soyuka** [comment]
Same without the `--raw` option?

**Comment by xiulunlin** [comment]
@soyuka  Yes. The app name is colored but raw data not.

<img width="612" alt="screen shot 2015-09-11 at 5 47 25 pm" src="https://cloud.githubusercontent.com/assets/7201022/9812036/4573d85e-58ad-11e5-9d54-1ec821af3626.png">

**Comment by soyuka** [comment]
@Tjatse thoughts on this? I try to find from where this issue could come, but we're directly streaming stdout output to the file. 

[Test case](https://gist.github.com/soyuka/f76116db91dc5d1d0447)

**Comment by chaudhryjunaid** [comment]
Yes, this issue exists even when using the --raw option. Strings in pretty printed objects are green, otherwise everything else including log level (info, warn, error) or custom colors using chalk.green, chalk.blue etc are not displayed.

**Comment by Tjatse** [comment]
So, you wanna enable colors?
Just try to use the following declaration / cmd to (re)start your `PM2` script:
```JSON
{
  "apps": [
    {
      "[SOME]":"[CONFIGS]",
      "args": [
        "--color"
      ]
    }
  ]
}
```
or
```bash
$ pm2 start myapp.js -- --color
```

**Comment by chaudhryjunaid** [comment]
works like a charm! thanks! :+1:

**Comment by damianobarbati** [comment]
Is this still valid @chaudhryjunaid ? Because I can see colors at all, no matter the --raw or --color combo (using config.js file).

**Comment by chaudhryjunaid** [comment]
@damianobarbati I have not used winston with pm2 in a long time now...
